### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by RamseyNjire**
Merge feature branch to master

Here we implemented the bubble sort algorithm. Any comments are welcome!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,25 @@
-# Project Name
+# Bubble Sort Algorithm
 
-> One paragraph statement about the project.
-
-![screenshot](./app_screenshot.png)
+> We implemented the bubble sort algorithm. Once to accept a simple array, and once with the ability to accept a code block supplied by the method caller.
 
 Additional description about the project and its features.
```
Comment (mosesogwo) [inline README.md]: You should remove this line if there is no need for additional description.

File: bubblesort.rb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+#set value of length as array.length - 1
+#[5, 4, 3, 2, 1]
+#[4, 5, 3, 2, 1]
+#change counter to true
+#[4, 3, 5, 2, 1]
+#changes counter to true
+#[4, 3, 2, 5, 1]
+#changes counter to true
+#[4, 3, 2, 1, 5]
+#changes counter to true
```
Comment (mosesogwo) [inline bubblesort.rb]: Remove all pseudo code and debugging instructions from this file.

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hello!:clap: Great job!. Please see my comments on your code**
Also, 
- Enable linter checks for this repository
- The ruby convention for file and method naming is snake_case, so you should rename your file and method as _bubble\_sort_

_**Kindly make these changes and submit another code review request.**_

**Any questions? You can reach me on Slack (Moses Ogwo)**

**Cheers:stars:**

### Status: Changes Requested.

**Comment by RamseyNjire** [comment]
I've implemented all you suggestions. However, it seems the linter won't work since it wasn't the first commit on master. I have included the files in both branches but it still doesn't work.

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi @RamseyNjire, @mimipeshy**
If we can't get this to work, then setup the linter in your local environment as described [here](https://github.com/microverseinc/linters-config/tree/master/ruby) and ensure you fix all the errors before submitting another code review request.

Your bubble_sort.rb file currently has a number of linter errors

Also, replace Author2 in your readme with the real name.


### Status: Changes required

**Review by nikasulo — APPROVED** [review]
# Great work @RamseyNjire 
Please merge and keep moving forward, I wish you the very best and see you at the top 🚀

**Pull request merged**